Makefile

```diff
@@ -111,7 +124,7 @@ integration_test: $(test_configs)
 
 .PHONY: $(test_configs)
 $(test_configs): ${driverkit}
-	${driverkit} docker -c $@ --builderimage falcosecurity/driverkit-builder:latest -l debug --timeout 600
+	${driverkit} docker -c $@ --builderimage auto:latest -l debug --timeout 600
```

Comment:
`auto:$imageTag` is also a supported builder image name now: it basically means "use automatic image selection but force an imageTag".